Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -6553,6 +6570,24 @@ These ItemGrids are available in the API and can be manipulated by the plugins,
 					},
 					Notes = "Adds the specified amount to the item count. Returns self (useful for chaining).",
 				},
+				AddEnchantmentsFromItem =
+				{
+					Params =
+					{
+						{
+							Name = "Sacrifice",
```

Comment:
I would avoid calling it the `Sacrifice`, since that would imply that this function destroys the item somehow. How about `Additive`, or `ToAdd`?